examples/with-next-lng/next.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  publicRuntimeConfig: {
```

Comment:
We no longer suggest usage of this field. Please switch this to the new environment support in Next.js 9.4!

Reply:
No more publicRuntimeConfig at all? I need to pass an array of languages at runtime, please kindly advise.

Reply:
You can use Next.js built in support for environment variables:
https://nextjs.org/docs/basic-features/environment-variables

Reply:
Thanks, I'll update asap.